lib/assessment.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
```

Comment:
Correct, for Ruby 2.0 and above; for older versions of Ruby, you need to include an `&` before the symbol.